assets/xml/objects/object_zl1.xml

```diff
@@ -1,90 +1,115 @@
-        <Texture Name="object_zl1_TLUT_000450" OutName="tlut_000450" Format="rgba16" Width="16" Height="16" Offset="0x450" />
-        <Texture Name="object_zl1_Tex_000650" OutName="tex_000650" Format="ci8" Width="8" Height="8" Offset="0x650" />
-        <Texture Name="object_zl1_Tex_000690" OutName="tex_000690" Format="ci8" Width="16" Height="16" Offset="0x690" />
-        <Texture Name="object_zl1_Tex_000790" OutName="tex_000790" Format="ci8" Width="32" Height="64" Offset="0x790" />
-        <Texture Name="object_zl1_Tex_000F90" OutName="tex_000F90" Format="ci8" Width="32" Height="8" Offset="0xF90" />
-        <Texture Name="object_zl1_Tex_001090" OutName="tex_001090" Format="ci8" Width="8" Height="8" Offset="0x1090" />
-        <Texture Name="object_zl1_Tex_0010D0" OutName="tex_0010D0" Format="ci8" Width="32" Height="32" Offset="0x10D0" />
-        <Texture Name="object_zl1_Tex_0014D0" OutName="tex_0014D0" Format="rgba16" Width="16" Height="16" Offset="0x14D0" />
-        <Texture Name="object_zl1_Tex_0016D0" OutName="tex_0016D0" Format="ci8" Width="16" Height="16" Offset="0x16D0" />
-        <Texture Name="object_zl1_Tex_0017D0" OutName="tex_0017D0" Format="rgba16" Width="8" Height="8" Offset="0x17D0" />
-        <Texture Name="object_zl1_Tex_001850" OutName="tex_001850" Format="rgba16" Width="16" Height="16" Offset="0x1850" />
-        <Texture Name="object_zl1_Tex_001A50" OutName="tex_001A50" Format="ci8" Width="8" Height="8" Offset="0x1A50" />
-        <Texture Name="object_zl1_Tex_001A90" OutName="tex_001A90" Format="ci8" Width="8" Height="8" Offset="0x1A90" />
-        <Texture Name="object_zl1_Tex_001AD0" OutName="tex_001AD0" Format="ci8" Width="16" Height="16" Offset="0x1AD0" />
-        <Texture Name="object_zl1_Tex_001BD0" OutName="tex_001BD0" Format="rgba16" Width="32" Height="32" Offset="0x1BD0" />
-        <DList Name="object_zl1_DL_004B50" Offset="0x4B50" />
-        <DList Name="object_zl1_DL_0054D0" Offset="0x54D0" />
-        <DList Name="object_zl1_DL_005738" Offset="0x5738" />
-        <DList Name="object_zl1_DL_005888" Offset="0x5888" />
-        <DList Name="object_zl1_DL_005A68" Offset="0x5A68" />
-        <DList Name="object_zl1_DL_005CD8" Offset="0x5CD8" />
-        <DList Name="object_zl1_DL_005E28" Offset="0x5E28" />
-        <DList Name="object_zl1_DL_006008" Offset="0x6008" />
-        <DList Name="object_zl1_DL_006828" Offset="0x6828" />
-        <DList Name="object_zl1_DL_0069E0" Offset="0x69E0" />
-        <DList Name="object_zl1_DL_006BA0" Offset="0x6BA0" />
-        <!-- <Texture Name="object_zl1_Tex_006CD8" OutName="tex_006CD8" Format="rgba16" Width="16" Height="16" Offset="0x6CD8" /> -->
-        <!-- <Blob Name="object_zl1_Blob_006ED8" Size="0x1F0" Offset="0x6ED8" /> -->
-        <!-- <Texture Name="object_zl1_TLUT_0070C8" OutName="tlut_0070C8" Format="rgba16" Width="16" Height="16" Offset="0x70C8" /> -->
-        <!-- <Blob Name="object_zl1_Blob_0072C8" Size="0x740" Offset="0x72C8" /> -->
```

Comment:
A lot of blobs seems to have been lost. Could you bring these comments back? There's more below as well